SpatialGDK/Source/SpatialGDK/Public/Interop/RPCs/RPCQueues.h

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Interop/RPCs/RPCService.h"
+
+namespace SpatialGDK
+{
+template <typename Payload, typename AdditionalSendingData = EmptyData>
+struct TRPCLocalOverflowQueue : public TRPCQueue<Payload, AdditionalSendingData>
```

Comment:
Local naming here isn't clear to me, could you explain?

Reply:
Guess i'm trying to distinguish it from the fact that the overflow could be stored remotely. It's poor naming, since it's really an unbounded queue.